src/website/views/components/alert.pug

```diff
@@ -99,29 +113,56 @@ p.-text
     each type in ['success', 'danger', 'warning', 'info', 'light', 'dark']
       .m-alert.-toast.-hasIcon.-m2(class=`-${type}`, data-title=`${type}`)
         = `This is a ${type} alert`
-        button(aria-label="Close")
+        button(class='a-btn -icon -close')
+          .a-btn__content
+            .a-icon
```

Comment:
remove duplicate `a-icon`. the `<i>` element has the only `a-icon` class we need